test/lint-charts.sh

```diff
@@ -86,19 +86,24 @@ validate_manifests() {
     echo "------------------------------------------------------------------------------------------------------------------------"
     for chart_name in ${changed_charts[*]} ; do
         echo "Validating chart ${chart_name}"
+        rm -rf "tmp/${chart_name}/"
+        mkdir -p tmp/stable
+        cp -r "${chart_name}" tmp/stable/
+        rm -rf tmp/"${chart_name}"/charts/
+        rm -rf tmp/"${chart_name}"/requirements.*
         echo "------------------------------------------------------------------------------------------------------------------------"
-        echo "==> Processing with ${REPO_ROOT}/${chart_name}/values.yaml"
+        echo "==> Processing with tmp/${chart_name}/values.yaml"
```

Comment:
@rimusz Why do we need this?

Reply:
because then kubeval validates and all charts from requirements.* files which fail.
we just need to validate our charts, as others we cannot fix

Reply:
@rimusz and why not just delete it from the REPO_ROOT? there's no git push in the CI process, right?